src/components/device/ha-device-automation-picker.ts

```diff
@@ -33,9 +33,13 @@ export abstract class HaDeviceAutomationPicker<
 
   @property() public value?: T;
 
-  protected NO_AUTOMATION_TEXT = "No automations";
+  protected NO_AUTOMATION_TEXT = this.hass?.localize(
+    "ui.panel.config.devices.automation.no_automations"
+  );
 
-  protected UNKNOWN_AUTOMATION_TEXT = "Unknown automation";
+  protected UNKNOWN_AUTOMATION_TEXT = this.hass?.localize(
```

Comment:
This is no longer used now?

Reply:
Currently not, but I was a bit hesitant to remove. Will get rid of it.

Reply:
You can leave it if you make the rest a getter, and make this a getter than too